voltcraft/pps.py

```diff
@@ -1,14 +1,21 @@
 """voltcraft.pps"""
+from __future__ import annotations
+
 import functools
 import sys
 
 import serial
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
 # The model can be identified by the maximum voltage and maximum current.
 # But this is probably one of the weirdest naming-schemes I've seen...
 # It just doesn't really make sense...
 PPS_MODELS = {
-    (18.0, 10.0): "PPS11810",  # not confirmed yet
+    ((18.0, 10.0)): "PPS11810",  # not confirmed yet
```

Comment:
Something strange must have happended here: why are there additional parentheses?

Reply:
Good question😬 Will remove it

Reply:
Done